mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -44,14 +44,14 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	FIELD field_5991 lastPortalPosition Lnet/minecraft/class_2338;
 	FIELD field_5992 verticalCollision Z
 	FIELD field_5993 pistonMovementDelta [D
-	FIELD field_5994 distanceWalked F
+	FIELD field_5994 distance F
```

Comment:
not super clear what this is? distance to what?

Reply:
I'll name this `distanceTravelled`. I changed it from "walked" since it's also counted for fish and birds.